libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams/Schemas/Actions/Teams.SendTabCardResponse.schema

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
+    "$role": "implements(Microsoft.IDialog)",
+    "title": "Tab card response",
+    "description": "Send n tab response containing cards.",
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "string",
+            "title": "Id",
+            "description": "Optional id for the dialog"
+        },
+        "cards": {
+            "$kind": "Microsoft.IActivityTemplate",
+            "title": "Cards",
+            "description": "Expession for Attachments template of Adaptive Cards to send. Attachment(s) is required."
```

Comment:
>Attachment(s) is required. [](start = 90, length = 26)

Why not put the property in required for the schema?

Reply:
Good catch!  went through and added required fields for all of the Teams .schema files